script/hassfest/config_flow.py

```diff
@@ -15,33 +17,50 @@
 FLOWS = {}
 """.strip()
 
+UNIQUE_ID_IGNORE = {"esphome", "fritzbox", "heos", "huawei_lte", "tradfri"}
```

Comment:
`tradfri` has been fixed and merged.

```suggestion
UNIQUE_ID_IGNORE = {"esphome", "fritzbox", "heos", "huawei_lte"}
```

Reply:
`esphome` is underway https://github.com/home-assistant/core/pull/34753